source/event_stream.c

```diff
@@ -18,6 +18,23 @@ static const char *AWS_EVENT_STREAM_PROPERTY_NAME_HEADERS = "headers";
 static const char *AWS_EVENT_STREAM_PROPERTY_NAME_PAYLOAD = "payload";
 static const char *AWS_EVENT_STREAM_PROPERTY_NAME_FLAGS = "flags";
 static const char *AWS_EVENT_STREAM_PROPERTY_NAME_MESSAGE = "message";
+static const char *AWS_EVENT_STREAM_PROPERTY_NAME_OPERATION = "operation";
+
+/*
+ * The fact that a zeroed array list crashes in aws_array_list_length drives me crazy.  Since CBMC proofs are
```

Comment:
ooc, would it be worthwhile to add these additional checks to a separate function in common that wouldn't crash for a zeroed array list? Or is that unnecessary clutter and better left to being checked in this way wherever we may have a zeroed array list?

Reply:
Unsure for now.  I would prefer to avoid touching array list in common due to the CBMC proofs and invariants.